test/cpp/end2end/grpclb_end2end_test.cc

```diff
@@ -442,7 +442,7 @@ class GrpclbEnd2endTest : public ::testing::Test {
 
   void WaitForBackend(size_t backend_idx) {
     do {
-      CheckRpcSendOk();
+      SendRpc();
```

Comment:
This is relaxed because now we have a situation where we need to keep checking while no backends are available for a long time and a call may fail due to timeout.